seedlet/assets/css/fonts.css

```diff
@@ -0,0 +1,40 @@
+@font-face {
+  font-family: 'Playfair Display';
+  font-display: swap;
+  font-style: normal;
+  font-weight: 100 400;
+  src: url("../fonts/Playfair_Display/static/PlayfairDisplay-Regular.ttf") format("truetype"); }
+
+@font-face {
+  font-family: 'Playfair Display';
+  font-display: swap;
+  font-style: bold;
+  font-weight: 500 900;
+  src: url("../fonts/Playfair_Display/static/PlayfairDisplay-Bold.ttf") format("truetype"); }
+
+@font-face {
+  font-family: 'Playfair Display';
+  font-display: swap;
+  font-style: italic;
+  src: url("../fonts/Playfair_Display/static/PlayfairDisplay-Italic.ttf") format("truetype"); }
+
+@font-face {
+  font-family: 'Fira Sans';
+  font-display: swap;
+  font-style: normal;
+  font-weight: 100 400;
+  src: url("../fonts/Fira_Sans/FiraSans-Regular.ttf") format("truetype"); }
+
+@font-face {
+  font-family: 'Fira Sans';
+  font-display: swap;
+  font-style: bold;
+  font-weight: 500 900;
+  src: url("../fonts/Fira_Sans/FiraSans-Bold.ttf") format("truetype"); }
+
+@font-face {
+  font-family: 'Fira Sans';
+  font-display: swap;
+  font-style: italic;
+  font-weight: 100 900;
```

Comment:
Should this be 400?: 
```suggestion
  font-weight: 100 400;
```

Reply:
I was trying to account for the editor allowing **_bold and italic_**.

However it looks like this combination isn't working, and we may have to include an additional font-face for declaration for the bold+italic use case: 

<img width="360" alt="Screen Shot 2020-08-04 at 5 51 09 PM" src="https://user-images.githubusercontent.com/5375500/89348788-25b28780-d67b-11ea-8023-569fc57019c3.png">